### PR TITLE
Make preCICE export paths consistent

### LINKED ISSUE #417: Inconsistent name of preCICE exports
_Originally posted by @davidscn in https://github.com/precice/tutorials/pull/411#discussion_r1410373649_

>  I think `preCICE-output` refers to the exports, right? Our configuration seems a bit arbitrary throughout the tutorials in this regard
```
./flow-over-heated-plate-nearest-projection/precice-config.xml:48:    <!-- <export:vtk directory="preCICE-output" /> -->
./replay-mode/precice-config.xml:24:         <export:vtu directory="exports" />
./two-scale-heat-conduction/precice-config.xml:53:      <export:vtu directory="precice-exports" />
./quickstart/precice-config.xml:41:    <export:vtk directory="coupling-meshes" />
```

### PR DESCRIPTION
**Pull request by MakisH**
I am renaming all directories in`<export:` tags to `precice-exports`, as currently suggested by our documentation.

- flow-over-heated-plate:
  - rename in config and README
  - enable exports by default, instead of the user having to explicitly add these (the case is small and short anyway)
  - check the visualization script and update the picture, including OpenFOAM-DuneFem. @BenjaminRodenberg why was that disabled before in the script, while we had it in the picture? Also, there were points used for OpenFOAM-FEniCS, but this is arbitrary in this context and does not originate from the script.
- flow-over-heated-plate-nearest-projection:
  - rename in (commented-out) config and README. I did not enable it, since it is not directly used by any script in this tutorial.
- quickstart:
  - rename in config and README
  - remove from the `clean.sh`. This is now covered by the `clean_precice_logs`.

I could not find any other relevant instances.

Closes #417.

### CODE REVIEW

**Comment by MakisH** [comment]
I have no idea why autopep8 complains in this PR, as I did not touch any Python files here.

**Review by uekerman — APPROVED** [review]
Good step

File: flow-over-heated-plate/plot-final-interface-temperature.py [hunk 1]
```diff
@@ -44,8 +44,9 @@ def vtk_to_dict(case):
 cases.append('fenics')
 cases.append('openfoam')
 cases.append('nutils')
+cases.append('dunefem')
 
-case_labels = {'fenics': 'OpenFOAM-FEniCS', 'openfoam': 'OpenFOAM-OpenFOAM', 'nutils': 'OpenFOAM-Nutils', }
+case_labels = {'fenics': 'OpenFOAM-FEniCS', 'openfoam': 'OpenFOAM-OpenFOAM', 'nutils': 'OpenFOAM-Nutils', 'dunefem': 'OpenFOAM-DuneFem'}
```
Comment (fsimonis) [inline flow-over-heated-plate/plot-final-interface-temperature.py:49]: This changes triggers the failure.

Comment (MakisH) [reply flow-over-heated-plate/plot-final-interface-temperature.py]: Looks like I was just plain blind, thanks!

**Pull request merged**